postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario2-Create payment with confirm false/Payments - Confirm/event.test.js

```diff
@@ -43,16 +43,4 @@ if (jsonData?.client_secret) {
    console.log("- use {{client_secret}} as collection variable for value",jsonData.client_secret);
 } else {
    console.log('INFO - Unable to assign variable {{client_secret}}, as jsonData.client_secret is undefined.');
-};
-
-// Response body should have value "succeeded" for "status"
```

Comment:
Why were these asserts removed?

Reply:
Unsure how these were removed. I only touched the requests.json for confirm calls. Added back.